packages/web-app-files/package.json

```diff
@@ -15,6 +15,7 @@
     "axios": "1.7.7",
     "design-system": "workspace:@ownclouders/design-system@*",
     "email-validator": "^2.0.4",
+    "escape-html": "^1.0.3",
```

Comment:
Please check if we can use `escape` from `lodash-es` instead of installing a new lib.